client/app/components/project/package-list-item.hbs

```diff
@@ -101,6 +101,56 @@
       {{/if}}
     </LinkTo>
   {{/if}}
+ {{#if (or
+    (eq @package.dcpPackagetype (optionset 'package' 'dcpPackagetype' 'code' 'DRAFT_EAS'))
+    (eq @package.dcpPackagetype (optionset 'package' 'dcpPackagetype' 'code' 'FILED_EAS'))
+  )}}
+    {{!-- TODO: When setting up Filed EAS, add filed-eas.edit and filed-eas.show as the false condition in route below
+    e.g.
+    (if (eq @package.dcpPackagetype (optionset 'package' 'dcpPackagetype' 'code' 'DRAFT_EAS')) 'draft-eas.edit' 'draft-eas.edit') --}}
+    <LinkTo
+      @route={{if (eq @package.statuscode (optionset 'package' 'statuscode' 'code' 'PACKAGE_PREPARATION'))
+        (if (eq @package.dcpPackagetype (optionset 'package' 'dcpPackagetype' 'code' 'DRAFT_EAS')) 'draft-eas.edit')
+        (if (eq @package.dcpPackagetype (optionset 'package' 'dcpPackagetype' 'code' 'DRAFT_EAS')) 'draft-eas.show')
+      }}
```

Comment:
I'm noticing this helper construction a lot. Could there be a simpler helper we could use to wrap this?

`{{if (has-option @package.statuscode 'package.statuscode.code.PACKAGE_PREPARATION')}}`

The second argument could use dot notation for object paths. I don't know off top of my head how the meta data istructured, but if it's nested, it could be as simple as using `get`:

`get(optionset, 'package.statuscode.code.PACKAGE_PREPARATION')` 

Just a thought.